src/components/vm/filesystems/vmFilesystemsCard.jsx

```diff
@@ -242,24 +241,6 @@ const VmFilesystemAddModal = ({ connectionName, memory, memoryBacking, objPath,
                                    onToggle={() => setAdditionalOptionsExpanded(!additionalOptionsExpanded)}
                                    isExpanded={additionalOptionsExpanded}>
                     <Grid hasGutter>
```

Comment:
This Grid with gutter is now unecessary since there is only one element inside it